Detectors/TRD/base/include/TRDBase/FeeParam.h

```diff
@@ -44,53 +45,51 @@ class FeeParam
 {
 
  public:
-  FeeParam(const FeeParam& p);
-  virtual ~FeeParam();
-  FeeParam& operator=(const FeeParam& p);
-  virtual void Copy(FeeParam& p) const;
+  ~FeeParam() = default;
+  FeeParam(const FeeParam&) = delete;
+  FeeParam& operator=(const FeeParam&) = delete;
 
   static FeeParam* instance(); // Singleton
-  static void terminate();
 
   // Translation from MCM to Pad and vice versa
-  static int getPadRowFromMCM(int irob, int imcm);
-  static int getPadColFromADC(int irob, int imcm, int iadc);
-  static int getExtendedPadColFromADC(int irob, int imcm, int iadc);
-  static int getMCMfromPad(int irow, int icol);
-  static int getMCMfromSharedPad(int irow, int icol);
-  static int getROBfromPad(int irow, int icol);
-  static int getROBfromSharedPad(int irow, int icol);
-  static int getROBSide(int irob);
-  static int getColSide(int icol);
```

Comment:
static functions are usually considered good (as they are stateless). Why taking them away here?

Reply:
They should not have been taken away you are right. I reverted those changes now